docs/intentReaderHandlers/HealthCheckHandler.md

```diff
@@ -0,0 +1,111 @@
+# HealthCheckHandler 
+
+## Overview
+
+HealthCheckHandler is invoked when the task specified in the intent has the value "healthCheck". This handler is used to check whether the FCA is successfully launched or not.
+
+It performs the following actions:
+1. Parses the input message received.
+2. Confirms the status of FCA, i.e., whether it is launched or not.
+3. Saves the response or error based on the status.
+4. Formats and sends the response back to IntentReader.
+
+## Usage
+
+### Request Format
+
+```json
+    {
+        "action": "search",
+        "data": {
+            "query": {
+                "task": "healthCheck",
+                "action": "NA",
+                "appType": "<appType>"
+            }
+        },
+        "context": {
+            "source": "device"
+        }
+    }
+```
+
+#### Parameters
+
+| Key               | Description                                                                       | Required? |
+|-------------------|-----------------------------------------------------------------------------------|-----------|
+| task              | "healthCheck"- It is a static value and should not be changed for this handler      | Y         |
+| appType           | Corresponding intent is launching on which app                                    | Y         |
+
+
+### Response Format
+* Response can be either "true" or an error response
```

Comment:
So just want to make sure this is accurate because in the example invalid response you have a "false" instead of a "error response". Is false an additional possibility or should it replace "error response"?
Please make sure the following things are all in sync with each other:
- `Response Format` description
- Response Parameters
- Example Responses

Please make sure these are aligned in all the handlers for both positive and negative scenarios, I think the majority are good already but have seen a few that looks out of place.